python/rikai/spark/utils.py

```diff
@@ -43,3 +44,34 @@ def get_default_jar_version(use_snapshot=True):
     if use_snapshot and (len(match_str) < len(version)):
         return match_str + "-SNAPSHOT"
     return match_str
+
+
+def init_spark_session(conf=None, app_name="rikai", rikai_version=None):
+    from pyspark.sql import SparkSession
+
+    if not rikai_version:
+        rikai_version = get_default_jar_version(use_snapshot=True)
+    builder = (
+        SparkSession.builder.appName(app_name)
+        .config(
+            "spark.jars.packages", "ai.eto:rikai_2.12:{}".format(rikai_version)
+        )
+        .config(
+            "spark.sql.extensions",
+            "ai.eto.rikai.sql.spark.RikaiSparkSessionExtensions",
+        )
+        .config(
+            "spark.driver.extraJavaOptions",
+            "-Dio.netty.tryReflectionSetAccessible=true",
+        )
+        .config(
+            "spark.executor.extraJavaOptions",
+            "-Dio.netty.tryReflectionSetAccessible=true",
+        )
+    )
+    conf = conf or {}
+    for k, v in conf.items():
+        builder = builder.config(k, v)
+    session = builder.master("local[2]").getOrCreate()
```

Comment:
is this user facing api? should we make `master` configuable?

Reply:
was just thinking about testing convenience and demo convenience. That being said, it could be a user facing API? what do you think? Would probably have to pull it up to a higher level module like `rikai.spark.init()` or something?

Reply:
Going to merge this for now. Can pull it up separately and parameterize master